webview.go

```diff
@@ -1,17 +1,18 @@
 package webview
 
 /*
-#cgo linux openbsd freebsd netbsd CXXFLAGS: -DWEBVIEW_GTK -std=c++11
+#cgo CXXFLAGS: -DWEBVIEW_EXPORTS -DWEBVIEW_STATIC -std=c++11
```

Comment:
This slipped through my initial checks. `WEBVIEW_EXPORTS` should be `WEBVIEW_BUILDING`. With that said, I am going to take another look later at this code for Go and whether using `WEBVIEW_STATIC` was intended.